pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -569,6 +569,13 @@ def __repr__(self):
         to_return = super().__repr__()
         return to_return.replace('given positions', 'each atom')
 
+    @property
+    def chemical_symbols(self):
+        """
+            Returns chemical symbols of the neighboring atoms.
```

Comment:
codacy is going to whine that this could be on one line. I don't actually care strongly about this formatting choice, but I do like making codacy shut up 😉

Reply:
Oh ok I didn't know that.